samples/tutorials/mnist/README.md

```diff
@@ -0,0 +1,87 @@
+# Kubeflow Pipeline Tutorial 
+[`Kubeflow Pipelines`](https://github.com/kubeflow/pipelines) is a platform for building and deploying portable, 
+scalable machine learning (ML) pipelines or workflows based on Docker containers. 
+The `Kubeflow Pipelines` platform consists of:
+- A user interface for managing and tracking experiments, jobs, and runs.
+- An engine for scheduling multi-step ML workflows.
+- An SDK for defining and manipulating pipelines and components.
+- Notebooks for interacting with the system using the SDK.
+
+A pipeline is a description of an ML workflow, including all of the components in the workflow and 
+how they combine in the form of a graph. The pipeline includes the definition of the inputs (parameters) required to 
+run the pipeline and the inputs and outputs of each component. A pipeline component is a self-contained set of user 
+code, packaged as a Docker image, that performs one step in the pipeline. For example, a component can be responsible 
+for steps such as data preprocessing, data transformation, and model training.
+
+## Content Overview:
+In this tutorial, we designed a series of notebooks to demonstrate how to interact with `Kubeflow Pipelines` through
+[Python SDK](https://github.com/kubeflow/pipelines/tree/master/sdk/python/kfp). In particular
```

Comment:
through Python SDK -> through the Kubeflow Pipelines SDK

Reply:
I have modified accordingly, and also removed the set up section from the readme.